### PR TITLE
Frontend UI

### PR DESCRIPTION
**Pull request by moizahmed102**
frontend of ecommerce app.

### CODE REVIEW

File: frontend/src/interceptors/axiosInstance.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+const axiosInstance = axios.create({
+  baseURL: "http://localhost:4000/api",
```
Comment (numan-ahmad) [inline frontend/src/interceptors/axiosInstance.js]: please use env here

File: frontend/src/interceptors/axiosInstance.js
Comment (numan-ahmad) [inline frontend/src/interceptors/axiosInstance.js:1]: please handle auth error here if user not authorize then move to login screen

File: frontend/src/services/authService.js
Comment (numan-ahmad) [inline frontend/src/services/authService.js:1]: please checkout route also add here if user not authorized then show login screen e.g when user select the product and then click to add cart after add then if he press the checkout button then he need to login first

File: frontend/src/interceptors/axiosInstance.js (hunk 1 above)
Comment (moizahmed102) [reply frontend/src/interceptors/axiosInstance.js]: resolved.

File: frontend/src/interceptors/axiosInstance.js
Comment (moizahmed102) [reply frontend/src/interceptors/axiosInstance.js:1]: resolved

File: frontend/src/services/authService.js
Comment (moizahmed102) [reply frontend/src/services/authService.js:1]: resolved

**Pull request merged**